app/Http/Livewire/TorrentSearch.php

```diff
@@ -783,6 +812,103 @@ final public function groupedPosters()
         return $groups;
     }
 
+    final public function alertConfirm(): void
+    {
```

Comment:
This block of code is repeated here, similar torrents, and torrent controller. It should be abstracted out.